staging/src/k8s.io/pod-security-admission/policy/check_allowPrivilegeEscalation.go

```diff
@@ -57,6 +57,12 @@ func CheckAllowPrivilegeEscalation() Check {
 }
 
 func allowPrivilegeEscalation_1_8(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) CheckResult {
+	// Pod API validation would have failed if podOS == Windows and if privilegeEscalation has been set.
+	// We can admit the Windows pod even if privilegeEscalation has not been set.
+	if podSpec.OS != nil && podSpec.OS.Name == corev1.Windows {
```

Comment:
Although nodes that don't understand pod os are technically out of the version skew window in my experience there is often a long tail of clusters running old nodes. I'm wondering if we should make these relaxed constraints a version bump. I.e. only the `v1.25` version of the AllowPrivilegeEscalation check would have this? Same for the capabilities & seccomp checks.

@liggitt WDYT?

Reply:
Are you thinking that clusters in that state would have pinned their Pod Security policies to 1.24 and earlier versions? If not, I don't really see what the version bump would accomplish

Reply:
Are you concerned that pod security admission can run as a webhook?

Reply:
Yeah, then this could have a release note like

> If you have nodes older than v1.23, namespaces using the restricted pod security level should version-pin to a version before v1.25

Reply:
Fair enough - I updated the release note. PTAL

Reply:
If we're going with that release note, then the policies also need to be updated to have a different implementation. Basically:

1. Add a `allowPrivilegeEscalation_1_25(...)` function, with this check (extract the common logic to a helper)
2. Update `CheckAllowPrivilegeEscalation()` to add a new version with the new function

Same for Capabilities & seccomp checks.